docs/next/guides/upgrade-and-migration/migrating-from-11.1-to-12.0.md

```diff
@@ -12,3 +12,225 @@ pageClass: migration-guide
+| ------- | --------------------------------------------------------------------------- | ----------------------------------------- |
+| Windows | <kbd>Ctrl</kbd> + <kbd>A</kbd> works<br><kbd>Cmd</kbd> + <kbd>A</kbd> works | Only <kbd>Ctrl</kbd> + <kbd>A</kbd> works |
+| macOS   | <kbd>Ctrl</kbd> + <kbd>A</kbd> works<br><kbd>Cmd</kbd> + <kbd>A</kbd> works | Only <kbd>Cmd</kbd> + <kbd>A</kbd> works  |
+
+::: tip
+The default keyboard shortcut for [merging cells](#keyboard-shortcuts-changes-cell-merging) is an exception: <kbd>Cmd</kbd> + <kbd>M</kbd> doesn't work anymore, as it conflicted with macOS's shortcut for window minimizing.
+:::
+
+#### Keyboard shortcuts changes: Navigation
+
+The table below summarizes default keyboard shortcuts changes related to navigation:
+
+| Windows                                   | macOS                                    | Before                                        | After                                                            |
+| ----------------------------------------- | ---------------------------------------- | --------------------------------------------- | ---------------------------------------------------------------- |
+| <kbd>**Ctrl**</kbd> + <kbd>**↑**</kbd>    | <kbd>**Cmd**</kbd> + <kbd>**↑**</kbd>    | Extend the selection by one cell up           | Move to the first cell of the current column                     |
+| <kbd>**Ctrl**</kbd> + <kbd>**↓**</kbd>    | <kbd>**Cmd**</kbd> + <kbd>**↓**</kbd>    | Extend the selection by one cell down         | Move to the last cell of the current column                      |
+| <kbd>**Ctrl**</kbd> + <kbd>**←**</kbd>    | <kbd>**Cmd**</kbd> + <kbd>**←**</kbd>    | Extend the selection by one cell to the left  | Move to the leftmost cell of the current row                     |
+| <kbd>**Ctrl**</kbd> + <kbd>**→**</kbd>    | <kbd>**Cmd**</kbd> + <kbd>**→**</kbd>    | Extend the selection by one cell to the right | Move to the rightmost cell of the current row                    |
+| <kbd>**Home**</kbd>                       | <kbd>**Home**</kbd>                      | Move to the first cell of the current row     | Move to the first non-frozen cell of the current row<sup>*</sup> |
+| <kbd>**Ctrl**</kbd> + <kbd>**Home**</kbd> | <kbd>**Cmd**</kbd> + <kbd>**Home**</kbd> | Move to the first cell of the current column  | Move to the first non-frozen cell of the grid<sup>*</sup>        |
+| <kbd>**End**</kbd>                        | <kbd>**End**</kbd>                       | Move to the last cell of the current row      | Move to the last non-frozen cell of the current row<sup>*</sup>  |
+| <kbd>**Ctrl**</kbd> + <kbd>**End**</kbd>  | <kbd>**Cmd**</kbd> + <kbd>**End**</kbd>  | Move to the last cell of the current column   | Move to the last non-frozen cell of the grid<sup>*</sup>         |
+
+<sup>*</sup> This action depends on your [layout direction](@/guides/internationalization/layout-direction.md).
```

Comment:
Should be plural ("these actions")

Reply:
@warpech - I don't think so: the asterisk refers to each action on its own (like a symbol in a map legend).